src/tools/cephfs/top/cephfs-top

```diff
@@ -591,10 +666,75 @@ class FSTop(object):
 
             stats_json = self.perf_stats_query()
             try:
-                if self.refresh_header(stats_json):
+                headline = "Headline: Selected Filesystem info [Press 'q' to see all filesystems!]"
```

Comment:
Suggestion: I feel `!` could be removed, just "Press 'q' to see all filesystems" looks better.

Reply:
Updated!!